core/src/main/resources/hudson/PluginManager/_table.css

```
@@ -7,7 +7,4 @@
 #filter-container {
     margin: 1em;
     text-align: right;
-}
-.compatWarning, .securityWarning {
-    font-weight: bold;
-}
+}
```

add line break at the end

```suggestion
}

```